arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -0,0 +1,2195 @@
+            "updatedAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "The time the namespace was updated."
+            },
+            "serviceBusEndpoint": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Endpoint you can use to perform Service Bus operations."
+            },
+            "metricId": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Identifier for Azure Insights metrics"
+            }
+          },
+          "description": "Properties of the Namespace."
+        }
+      },
+      "description": "Description of a Namespace resource.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ]
+    },
+	"RelayNamespacePatch": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Sku of the Namespace."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "properties": {
+            "provisioningState": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Provisioning state of the namespace."
+            },
+            "createdAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "The time the namespace was created."
+            },
+            "updatedAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "The time the namespace was updated."
+            },
+            "serviceBusEndpoint": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Endpoint you can use to perform Service Bus operations."
+            },
+            "metricId": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Identifier for Azure Insights metrics"
+            }
+          },
+          "description": "Properties of the Namespace."
+        }
+      },
+      "description": "Description of a Namespace resource.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResourcePatch"
+        }
+      ]
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of this Sku",
+          "enum": [
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": false
+          }
+        },
+        "tier": {
+          "type": "string",
+          "description": "The tier of this particular SKU",
+          "enum": [
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": false
```

Comment:
Yes, we want to expose it as an enum to people consuming the SDK so that they can strong type it